include/re_sa.h

```diff
@@ -27,6 +35,7 @@ struct sa {
 	union {
 		struct sockaddr sa;
 		struct sockaddr_in in;
+		struct sockaddr_un un;
```

Comment:
I wonder if we should make this compile optional? like "HAVE_UNIXSOCK", it increases `struct sa` size by 108 bytes.

Reply:
I would prefer positive logic here, i.e. ifdef HAVE_UNIXSOCK

also please note that if libre and application is compiled with different flags,
the size of struct sa would be different, and lead to crash.